interp/notation.ml

```diff
@@ -143,7 +144,9 @@ let normalize_scope sc =
     try
       let sc = String.Map.find sc !delimiters_map in
       let _ = String.Map.find sc !scope_map in sc
-    with Not_found -> error_unknown_scope sc
+    with Not_found as exn ->
```

Comment:
Noise.

Reply:
Is that really noise, without this info about which `find` failed will be missing.